OSPSuite.Presentation.Importer/Views/IUnitsEditorView.cs

```diff
@@ -0,0 +1,16 @@
+﻿using OSPSuite.Core.Domain.UnitSystem;
+using OSPSuite.Presentation.Importer.Presenters;
+using OSPSuite.Presentation.Views;
+using System.Collections.Generic;
+
+namespace OSPSuite.Presentation.Importer.Views
+{
+   public interface IUnitsEditorView : IModalView<IUnitsEditorPresenter>
+   {
+      void SetParams(bool useDimensionSelector);
+
```

Comment:
sometimes there are space between method names, sometimes not. Let's be consistent

Reply:
spaces removed from methods where they existed (not from event declarations, for better readability).